src/metadata.rs

```diff
@@ -6,23 +6,33 @@ use byteorder::{LittleEndian, ReadBytesExt, WriteBytesExt};
 use crate::compression::CompressionType;
 use crate::error::Error;
 
-const METADATA_SIZE: usize = 17;
+const METADATA_V1_SIZE: usize = 17;
+const METADATA_V2_SIZE: usize = 18;
 const MAGIC_V1: u32 = 0x76324D4C;
+const MAGIC_V2: u32 = 0x6723D4C4;
```

Comment:
👀 🔀

Reply:
Yup, that's just a dump number. I just changed the magic number a little bit.